src/lib/TranscriptEditor/index.js

```diff
@@ -98,26 +134,59 @@ class TranscriptEditor extends React.Component {
 
   handleShowTimecodes = (e) => {
     const isChecked = e.target.checked;
-
     this.setState({ showTimecodes: isChecked });
+
+    if (this.props.handleAnalyticsEvents !== undefined) {
+      this.props.handleAnalyticsEvents({ 
+        category: 'TranscriptEditor', 
+        action: 'handleShowTimecodes', 
+        name: 'showTimecodes', 
+        value: isChecked
+      });
+    }
   }
 
   handleShowSpeakers = (e) => {
     const isChecked = e.target.checked;
-
     this.setState({ showSpeakers: isChecked });
+
+    if (this.props.handleAnalyticsEvents !== undefined) {
+      this.props.handleAnalyticsEvents({ 
+        category: 'TranscriptEditor', 
+        action: 'handleShowSpeakers', 
+        name: 'showSpeakers', 
+        value:  isChecked
```

Comment:
remove extra space (and do the same for the next two)

Reply:
ok done